selenium_tests/course_info/page_objects/course_info_search_page_object.py

```diff
@@ -20,8 +20,14 @@ class Locators(object):
     @classmethod
     def COURSE_LINK_HREF_CSS(cls, cid):
         """ returns a locator for a course detail link in the course table """
-        return By.CSS_SELECTOR, 'a[href="#/people/{}"]'.format(cid)
+        return By.CSS_SELECTOR, 'a[href="#/details/{}"]'.format(cid)
 
+    @classmethod
+    def COURSE_CODE_TEXT(cls, course_code_text):
```

Comment:
This doesn't match the course code column -- this finds text in any cell in the table. It can be generalized to TD_TEXT(cls, text_to_match_in_td) or something like that. The course people PO has the locator `TD_TEXT_XPATH(cls, search_text)`; this is the same thing. In fact, you could move `TD_TEXT_XPATH()` into the course info base PO so it isn't defined twice.